Lexer/TokenRepository.cs

```diff
@@ -0,0 +1,276 @@
+using System.Security.Principal;
+using Common;
+using Lexer.Enums;
+using Lexer.Enums;
+using Lexer.Tokens;
+
+namespace Lexer
+{
+    public class TokenRepository
+    {
+        public static Token MakeNumber(Logger Logger)
+        {
+            var numberString = string.Empty; 
+            var dotCount = 0;
+
+            while (Lexer.currentChar != null && (Globals.DIGITS.Contains(Lexer.currentChar) || Lexer.currentChar == '.'))
+            {
+                if (Lexer.currentChar == '.')
+                {
+                    if (dotCount == 1)
+                    {
+                        break;
+                    }
+                    dotCount += 1;
+                    numberString += '.';
+                }
+                else
+                {
+                    numberString += Lexer.currentChar;
+                }
+                Lexer.Advance();
+            }
+
+            if (dotCount == 0)
+            {
+                return new Token(TokenGroup.VALUE, TokenValue.INT, Logger, int.Parse(numberString));
+            }
+            else
+            {
+                return new Token(TokenGroup.VALUE, TokenValue.FLOAT, Logger, float.Parse(numberString));
+            }
+        }
+
+        public static Token MakeIdentifier(Logger Logger)
+        {
+            var idString = string.Empty;
+
+            while (Lexer.currentChar != null && (Globals.LETTERS_DIGITS.Contains(Lexer.currentChar) || Lexer.currentChar == '_'))
+            {
+                idString += Lexer.currentChar;
+                Lexer.Advance();
+            }
+
+            var tokenType = new object();
+            var typeKeywords = new string[] {"var", "int", "float", "string", "char", "new"};
+            var controlKeywords = new string[] {"if", "else", "for", "to", "foreach", "in", "while", "func", "class"};
+
+            if (typeKeywords.Contains(idString))
```

Comment:
Make a typeKeyWords manager which handles this

Reply:
#47